pdb-alignment-pipeline/src/main/java/org/cbioportal/pdb_annotation/scripts/PdbScriptsPipelineMakeSQL.java

```diff
@@ -0,0 +1,389 @@
+package org.cbioportal.pdb_annotation.scripts;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Unmarshaller;
+
+import org.apache.commons.io.FileUtils;
+import org.cbioportal.pdb_annotation.util.ReadConfig;
+import org.cbioportal.pdb_annotation.util.blast.BlastDataBase;
+import org.cbioportal.pdb_annotation.util.blast.BlastOutput;
+import org.cbioportal.pdb_annotation.util.blast.BlastOutputIterations;
+import org.cbioportal.pdb_annotation.util.blast.BlastResult;
+import org.cbioportal.pdb_annotation.util.blast.Hit;
+import org.cbioportal.pdb_annotation.util.blast.Hsp;
+import org.cbioportal.pdb_annotation.util.blast.Iteration;
+import org.cbioportal.pdb_annotation.util.blast.IterationHits;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * SQL Insert statments Generation
+ * 
+ * @author Juexin Wang
+ *
+ */
+@Component
+public class PdbScriptsPipelineMakeSQL {
+	private BlastDataBase db;
+	private int matches;
+	private int ensembl_file_count;	
```

Comment:
Camel case is preferred over underscore when naming Java variables. So, `ensemblFileCount` is preferred over `ensembl_file_count`